ghga_service_chassis_lib/object_storage_dao_testing.py

```diff
@@ -70,6 +72,58 @@ def upload_file(presigned_url: PresignedPostURL, file_path: Path, file_md5: str)
         response.raise_for_status()
 
 
+def calc_part_size(file_path: Path, n_parts: int) -> int:
+    """Calculate the part size in bytes for the given file when desiring the specified number of
+    parts.
+    Please note, the part size must be at least 25 min"""
+    file_size = os.path.getsize(file_path)
+    return max(math.ceil(file_size / n_parts), 50 * pow(10, 6))
```

Comment:
Do you mean 25 MiB?
Part size must be 5 MiB for S3, though.
Also, 50 * pow(10, 6) is neither 5 MiB, nor 5 MB

Reply:
That's of course correct.